src/main/java/cuchaz/enigma/translation/mapping/serde/TinyV2Writer.java

```diff
@@ -59,7 +59,7 @@ private void writeClass(PrintWriter writer, EntryTreeNode<EntryMapping> node, En
 			if (mapping != null) {
 				parts.addFirst(mapping.getTargetName());
 			} else {
-				parts.addFirst(classEntry.getFullName());
```

Comment:
Why not just omit the call

Reply:
wouldn't that mean we lose the trailing tab?

Reply:
Even when you have tabs, this means you are literally declaring `""` as the name in those trailing namespaces, which would make the thing worse